src/Core/Grid/LinkBlockGridFactory.php

```diff
@@ -86,6 +94,11 @@ public function getGrids(array $hooks, array $filtersParams)
         foreach ($hooks as $hook) {
             $hookParams = $filtersParams;
             $hookParams['filters']['id_hook'] = $hook['id_hook'];
+            $hookParams['filters']['id_shop'] = [$this->shopContext->getContextShopID()];
+            
+            if (!$this->shopContext->isSingleShopContext()) {
```

Comment:
As we have possibility to sort link blocks, I think it would be much better to allow people to edit this module only in single store context 🤔 

what do you think @matthieu-rolland ?

Reply:
I think this is to be discussed with the product team, but I believe you and @marionf talked about it, what came out it?

Reply:
I should have an answer on Thursday